FeedAPIChallenge/Feed API/RemoteFeedLoaderMapper.swift

```diff
@@ -0,0 +1,33 @@
+
+import Foundation
+
+internal final class RemoteFeedImagesMapper {
```

Comment:
`internal` is the default access level so you can omit it if you want to (we type it explicitly in the lectures for teaching purposes, but you don't need to type it in your code).

You can remove the `internal` keyword from anywhere used.

```suggestion
final class RemoteFeedImagesMapper {
```